frontend/app/routes/public/apply/$id/adult/confirm-federal-provincial-territorial-benefits.tsx

```diff
@@ -0,0 +1,202 @@
+  const { t } = useTranslation(handle.i18nNamespaces);
+  const { defaultState, editMode } = loaderData;
+
+  const fetcher = useFetcher<typeof action>();
+  const isSubmitting = fetcher.state !== 'idle';
+  const [federalProvincialTerritorialBenefitChangedValue, setFederalProvincialTerrirorialBenefitChangedValue] = useState(defaultState);
+
+  const errors = fetcher.data?.errors;
+  const errorSummary = useErrorSummary(errors, {
+    hasFederalProvincialTerritorialBenefits: 'input-radio-federal-provincial-territorial-benefits-changed-option-0',
+  });
+
+  function handleOnFederalProvincialTerritorialBenefitChanged(e: React.ChangeEvent<HTMLInputElement>) {
+    setFederalProvincialTerrirorialBenefitChangedValue(e.target.value === FEDERAL_BENEFITS_CHANGED_OPTION.yes);
+  }
+
+  return (
+    <>
+      <div className="my-6 sm:my-8">
+        <Progress value={88} size="lg" label={t('apply:progress.label')} />
+      </div>
+      <div className="max-w-prose">
+        <p className="mb-4">{t('apply-adult:confirm-dental-benefits.access-to-dental')}</p>
+        <p className="mb-4">{t('apply-adult:confirm-dental-benefits.eligibility-criteria')}</p>
+        <p className="mb-4 italic">{t('apply:required-label')}</p>
+        <errorSummary.ErrorSummary />
+        <fetcher.Form method="post" noValidate>
+          <CsrfTokenInput />
+          <fieldset className="mb-6">
+            <InputRadios
+              id="federal-provincial-territorial-benefits-changed"
+              name="hasFederalProvincialTerritorialBenefits"
+              legend={t('apply-adult:confirm-dental-benefits.has-benefits')}
+              options={[
+                {
+                  children: <Trans ns={handle.i18nNamespaces} i18nKey="apply-adult:confirm-dental-benefits.option-yes" />,
+                  value: FEDERAL_BENEFITS_CHANGED_OPTION.yes,
+                  defaultChecked: federalProvincialTerritorialBenefitChangedValue === true,
+                  onChange: handleOnFederalProvincialTerritorialBenefitChanged,
+                },
+                {
+                  children: <Trans ns={handle.i18nNamespaces} i18nKey="apply-adult:confirm-dental-benefits.option-no" />,
+                  value: FEDERAL_BENEFITS_CHANGED_OPTION.no,
+                  defaultChecked: federalProvincialTerritorialBenefitChangedValue === false,
+                  onChange: handleOnFederalProvincialTerritorialBenefitChanged,
+                },
+              ]}
+              errorMessage={errors?.hasFederalProvincialTerritorialBenefits}
+              required
+            />
+          </fieldset>
+          {editMode ? (
+            <div className="mt-8 flex flex-wrap items-center gap-3">
+              <Button id="save-button" variant="primary" disabled={isSubmitting} data-gc-analytics-customclick="ESDC-EDSC:CDCP Apply Application Form-Adult:Save - Access to other federal, provincial or territorial dental benefits click">
```

Comment:
Just to match the title of the page (same comment for other `data-gc-analytics-customclick` values below):

```suggestion
              <Button id="save-button" variant="primary" disabled={isSubmitting} data-gc-analytics-customclick="ESDC-EDSC:CDCP Apply Application Form-Adult:Save - Access to other government dental benefits click">
```

Reply:
Fixed the analytics data/string